src/main/resources/application.yml

```diff
@@ -6,3 +6,7 @@ chs:
   kafka:
     api:
       endpoint: ${CHS_KAFKA_API_URL:localhost}
+
+springfox:
+  documentation:
+    auto-startup: false
```

Comment:
May cause issues when you bump up the `private-api-sdk` version

Reply:
Should be removed once the private sdk version is updated to latest version.